src/blocks/services/index.js

```diff
@@ -6,48 +6,184 @@ import { ServicesIcon as icon } from '@godaddy-wordpress/coblocks-icons';
 /**
  * Internal dependencies.
  */
-import deprecated from './deprecated';
-import edit from './edit';
-import example from './example';
 import metadata from './block.json';
-import save from './save';
 
 /**
  * WordPress dependencies.
  */
 import { __ } from '@wordpress/i18n';
 import { Icon } from '@wordpress/components';
+import { InnerBlocks } from '@wordpress/block-editor';
+import { createBlock, switchToBlockType } from '@wordpress/blocks';
+import { useDispatch, useSelect } from '@wordpress/data';
 
 /**
  * Block constants.
  */
-const { name, category, attributes } = metadata;
+const { name, category } = metadata;
+
+const SERVICE_TEMPLATE = [
+	[
+		'core/image',
+		{
+			align: 'full',
+			allowResize: false,
+			alt: '',
+			className: 'is-style-service',
+		},
+	],
+	[
+		'core/heading',
+		{
+			/* translators: content placeholder */
+			content: __( '', 'coblocks' ),
+			level: 4,
+			/* translators: content placeholder */
+			placeholder: __( 'Write title…', 'coblocks' ),
+			textAlign: 'center',
+		},
+	],
+	[
+		'core/paragraph',
+		{
+			align: 'center',
+			/* translators: content placeholder */
+			content: __( '', 'coblocks' ),
+			/* translators: content placeholder */
+			placeholder: __( 'Write description…', 'coblocks' ),
+		},
+	],
+];
+
+const templateContainer = [
+	SERVICE_TEMPLATE,
+	SERVICE_TEMPLATE,
+];
+
+function Edit( { clientId } ) {
+	const { replaceBlocks } = useDispatch( 'core/block-editor' );
+	const { getBlock } = useSelect( ( select ) => select( 'core/block-editor' ) );
+
+	const currentBlock = getBlock( clientId );
+
+	if ( ! currentBlock ) {
+		return null;
+	}
+
+	replaceBlocks(
+		[ clientId ],
+		switchToBlockType( currentBlock, 'core/columns' )
+	);
+
+	return null;
+}
+
+const migrateCurrent = ( attributes, innerBlocks ) => {
+	return innerBlocks.map( ( innerBlock ) => switchToBlockType( innerBlock, 'core/column' ) ).flat();
+};
+
+const migrateNew = () => {
+	return templateContainer.map( ( innerBlock ) => {
+		return createBlock(
+			'core/column',
+			{},
+			innerBlock.map( ( block ) => {
+				return createBlock(
+					...block,
+				);
+			} )
+		);
+	} );
+};
+
+const formatColumns = ( innerBlocks, colAmount ) => {
```

Comment:
since we are mutating a version of the `innerBlocks` array, it might be better to clone, mutate, and return. rather than mutating directly which could mess with affect the original innerBlocks array. I think it's better to clone when doing these types of operations as a defensive coding standard but in this case the original innerBlocks array could be mutate even outside of this scope without cloning it first.